app/views/insured/product_shoppings/show.html.slim

```diff
@@ -377,8 +387,13 @@ javascript:
       `
       <div class="module plan-row panel panel-default" data-cuke="filtered-plan">
               <div class="panel-body">
-                <div class="row ${result.product.is_standard_plan? '' : 'hidden'}">
-                  <span class="ml-1 text-uppercase" style="color:green"><i class="fas fa-bookmark"></i> Standard Plan</span>
+                <div class="row">
+                  <span class=" ${result.is_pvp_eligible? '' : 'hidden'} pvp-plan-indicator ml-1">
```

Comment:
Might be a bit more clear/grep-friendly if this was 
```result.is_pvp_eligible ? ''...```
 instead of 
```result.is_pvp_eligible? ''...```

This looks like an invocation of an existing ruby method, but is just unorthodox JS formatting. (Totally understanding this is what was there before.)